pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -160,6 +184,37 @@ func (a *Operator) syncOperatorGroups(obj interface{}) error {
 	return nil
 }
 
+func unique(logger *logrus.Entry, a, b []string) []string {
+	presentInANotInB := []string{}
+	// Check if either list is is empty to save time.
+	// If the list is of length 1, we much check that the namespace is not an
+	// empty string because the status object defaults to the following:
+	// status:
+	//   namespaces:
+	//   - ""
```

Comment:
If the namespaces list is set `""`, it means `AllNamespaces`. So does that mean you need to add the OG label to all namespaces that OLM is watching?

Reply:
In this case I would not apply a label.